cypress/support/commands.js

```diff
@@ -506,3 +506,18 @@ Cypress.Commands.add("verifyErrorSummaryTitle", () => {
 Cypress.Commands.add("verifyErrorMessage", (errorMessage) => {
 	cy.get("li > a").should("exist").should("have.text", errorMessage);
 });
+
+Cypress.Commands.add("verifyExternalLinkResponse", (externalLinkText) => {
+	cy.contains("a", externalLinkText)
+		.invoke("attr", "href")
+		.then(function (val) {
+			cy.request({
+				url: val,
```

Comment:
Are we not calling out to an external service here? we should avoid that or if that service becomes unavailable we won't be able to deploy

Reply:
Resolved